container-metrics.html.md.erb

```diff
@@ -126,3 +130,28 @@ Showing CPU usage against entitlement for app dora-example in org example-org /
 ​
 </pre>
 
+### <a id="cf-app"></a> The CPU Overentitlement Plugin
+
+The CPU overentitlement plugin shows when an app has been throttled. This information helps determine if the app needs to be scaled.
+
+The avarage usage from the CPU entitlement plugin is used to split the application into 2 groups - good and bad apps. Good apps are those that have an average CPU usage under 100% of their CPU entitlement, respectively bad apps are those whose average CPU usage exceeds 100% of their entitlements.
+
+You can check all the so called bad applications in your organization by using the CPU overentitlement plugin and the `cf over-entitlement-instances` command.
+
+1. Install the CPU Overentitlement Plugin from [cpu-entitlement-plugin](https://github.com/cloudfoundry/cpu-entitlement-plugin) in GitHub.
+
+2. Run the command `cf over-entitlement-instances`.
+
+For example:
+
+<pre class="terminal">
+$ cf over-entitlement-instances
+Note: This feature is experimental.
+Showing over-entitlement apps in org example-org / space example-org-staging as dora@example.com ...
+​
+     space                     app
+#0   example-org-staging       dora-example-2     
+#1   example-org-staging       dora-example-3
+#2   example-org-staging       dora-example-4
+​
+</pre>
```

Comment:
❓ Maybe add a wrap a sentence or something. Like: this means that these 3 apps are have an average CPU usage that exceeds 100% of their entitlements. Operators should consider doing XYZ with this info.